src/main/java/pages/SignUpPage.java

```diff
@@ -0,0 +1,78 @@
+package pages;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SignUpPage extends BasePage {
+    @FindBy(xpath = "//div[@class='container page']//h1[text()='Sign Up']")
+    private WebElement titleSingUpPage;
+
+    @FindBy(xpath = "//input[@placeholder='Username']")
+    private WebElement inputUserName;
+    @FindBy(xpath = "//input[@placeholder='Email']")
+    private WebElement inputEmail;
+    @FindBy(xpath = "//input[@placeholder='Password']")
+    private WebElement inputPassword;
+    @FindBy(xpath = "//button[@type='submit']")
+    private WebElement buttonSingUp;
+
+    @FindBy(className = "error-messages")
+    private WebElement panelErrorMsg;
+    @FindBy(xpath = "//ul[@class='error-messages']/li")
+    private List<WebElement> listErrorMsg;
```

Comment:
Leave an empty line between every property declaration, just like in lines 13 and 22

Reply:
Blanks are removed

Reply:
😅 I was telling the opposite: to add a blank line between every property. It's ok by now, no problem 👍🏽

Reply:
:) it is understood